### PR TITLE
nrf_rpc: applications: Move rpc UART transport to new work queue.

### PR DESCRIPTION
**Pull request by canisLupus1313**
Commit moves to NRF RPC uart transport to use it's own work queue.

### CODE REVIEW

File: subsys/nrf_rpc/nrf_rpc_uart.c [hunk 1]
```diff
@@ -106,10 +107,12 @@ static void serial_cb(const struct device *uart, void *user_data)
 			break;
 		}
 
-		k_work_submit(&uart_tr->cb_work);
+		k_work_submit_to_queue(&uart_tr->rx_workq, &uart_tr->cb_work);
 	}
 }
 
+K_THREAD_STACK_DEFINE(uart_tr_workq_stack_area, 4096);
```
Comment (Damian-Nordic) [inline subsys/nrf_rpc/nrf_rpc_uart.c:114]: Worth adding TODO: for moving it to uart_tr.

File: subsys/nrf_rpc/nrf_rpc_uart.c [hunk 2]
```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#include "zephyr/kernel.h"
```
Comment (Damian-Nordic) [inline subsys/nrf_rpc/nrf_rpc_uart.c:7]:
```suggestion
#include <zephyr/kernel.h>
```
super nit but other includes in this file follow a different style

**Review by Damian-Nordic — APPROVED** [review]

File: subsys/nrf_rpc/nrf_rpc_uart.c (hunk 1 above)
Comment (canisLupus1313) [reply subsys/nrf_rpc/nrf_rpc_uart.c:114]: done

File: subsys/nrf_rpc/nrf_rpc_uart.c (hunk 2 above)
Comment (canisLupus1313) [reply subsys/nrf_rpc/nrf_rpc_uart.c]: Moved to header file.

**Review by bpienk — APPROVED** [review]

**Review by Damian-Nordic — APPROVED** [review]

**Pull request merged**